### PR TITLE
[RFC] pipewire, wireplumber, pulseaudio: move system services to /usr/share/examples/PKGNAME/sv/

### PR DESCRIPTION
**Pull request by classabbyamp**
To reduce confusion and counteract users who enable every service they see willy-nilly, hide the system services for pipewire, pipewire-pulse, wireplumber, and pulseaudio in `/usr/share/examples/$pkgname/sv/`.

originally suggested by @0x5c in IRC

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Review by abenson — APPROVED** [review]

**Comment by ahesford** [comment]
This is fine, but I think it might be better to completely remove the pipewire-pulse and wireplumber services, modify the (now example) pipewire service to wrap it in `dbus-run-session`, and tell people that want to use it that they should make pipewire launch pipewire-pulse and wireplumber via the proposed example config snippets.

This probably requires `control/t` and the pgrphack (`chpst -P`) trick in the current wireplumber service to make a best effort to kill all children.

This is essentially how I run a pipewire system service, and it is much more reliable and straightforward than trying to manage multiple services.

**Comment by classabbyamp** [comment]
how's this look? I haven't tested it because I don't run pw that way.

Does `/run/pulse` still need creation if missing?

File: srcpkgs/pipewire/files/README.voidlinux [hunk 1]
```diff
@@ -8,6 +8,7 @@ Optional dependencies:
 * `libspa-v4l2` for video camera support
 * `rtkit` for realtime priority acquisition
 
-The system services /etc/sv/pipewire and /etc/sv/pipewire-pulse are
-experimental and only needed in rare cases so using them should be
-avoided in most setups.
+The example system service /usr/share/examples/pipewire/sv/pipewire is
+experimental and only needed in rare cases so using it should be avoided
```
Comment (ahesford) [inline srcpkgs/pipewire/files/README.voidlinux:12]:
```suggestion
experimental and only needed in rare cases, so using it should be avoided
```

File: srcpkgs/pipewire/files/README.voidlinux [hunk 2]
```diff
@@ -8,6 +8,7 @@ Optional dependencies:
...
+in most setups. To use it, pipewire needs to be configured to launch
+pipewire-pulse and wireplumber as written in the documentation.
```
Comment (ahesford) [inline srcpkgs/pipewire/files/README.voidlinux:14]:
```suggestion
pipewire-pulse and wireplumber as documented at

    https://docs.voidlinux.org/config/media/pipewire.html
```

**Review by ahesford — APPROVED** [review]
I think the current proposal greatly improves the pipewire service even as we discourage its use.

**Pull request merged**